Cubical/Data/Int/MoreInts/QuoInt/Properties.agda

```diff
@@ -233,3 +236,84 @@ sign-pos-suc-· m (signed s (suc n)) = refl
+
+  sucℤ→Int : ∀ (n : ℤ) → Int.sucℤ (ℤ→Int n) ≡ ℤ→Int (sucℤ n)
+  sucℤ→Int (pos n) = refl
+  sucℤ→Int (neg zero) = refl
+  sucℤ→Int (neg (suc zero)) = refl
+  sucℤ→Int (neg (suc (suc n))) = refl
+  sucℤ→Int (posneg i) = refl
+
+  predℤ→Int : ∀ (n : ℤ) → Int.predℤ (ℤ→Int n) ≡ ℤ→Int (predℤ n)
+  predℤ→Int (pos zero) = refl
+  predℤ→Int (pos (suc n)) = refl
+  predℤ→Int (neg zero) = refl
+  predℤ→Int (neg (suc n)) = refl
+  predℤ→Int (posneg i) = refl
+
+  ℤ→Int+Int≡+ : ∀ (n m : ℤ) → (ℤ→Int n) Int.+ (ℤ→Int m) ≡ ℤ→Int (n + m)
+  ℤ→Int+Int≡+ n m = (ℤelim (λ n → ∀ (m : ℤ) → (ℤ→Int n) Int.+ (ℤ→Int m) ≡ ℤ→Int (n + m)) posℤ→Int+Int≡+ negsucℤ→Int+Int≡+ n) m
+    where
+    posℤ→Int+Int≡+ : ∀ (n : ℕ) (m : ℤ) → (ℤ→Int (pos n)) Int.+ (ℤ→Int m) ≡ ℤ→Int ((pos n) + m)
+    posℤ→Int+Int≡+ zero m =
+      (ℤ→Int (pos zero)) Int.+ (ℤ→Int m)
+        ≡⟨ sym (Int.pos0+ (ℤ→Int m)) ⟩
+      ℤ→Int m
+        ≡⟨ cong ℤ→Int (sym (+-zeroʳ spos m)) ⟩
+      ℤ→Int (m + pos zero)
+        ≡⟨ cong ℤ→Int (+-comm m (pos zero)) ⟩
+      ℤ→Int (pos zero + m) ∎
+    posℤ→Int+Int≡+ (suc n) m =
+      (ℤ→Int (pos (suc n))) Int.+ (ℤ→Int m)
+        ≡⟨ sym (Int.sucℤ+ (Int.pos n) (ℤ→Int m)) ⟩
+      Int.sucℤ ((Int.pos n) Int.+ (ℤ→Int m))
+        ≡⟨ cong Int.sucℤ (posℤ→Int+Int≡+ n m) ⟩
+      Int.sucℤ (ℤ→Int ((pos n) + m))
+        ≡⟨ sucℤ→Int ((pos n) + m) ⟩
+      ℤ→Int (sucℤ ((pos n) + m))
+        ≡⟨ cong ℤ→Int (sucℤ-+ˡ (pos n) m) ⟩
+      ℤ→Int ((pos (suc n)) + m) ∎
+
+    negsucℤ→Int+Int≡+ : ∀ (n : ℕ) (m : ℤ) → (ℤ→Int (neg (suc n))) Int.+ (ℤ→Int m) ≡ ℤ→Int ((neg (suc n)) + m)
+    negsucℤ→Int+Int≡+ zero m =
+      Int.negsuc zero Int.+ ℤ→Int m
+        ≡⟨ sym (Int.predℤ+ (Int.pos zero) (ℤ→Int m)) ⟩
+      Int.predℤ ((Int.pos zero) Int.+ (ℤ→Int m))
+        ≡⟨ cong Int.predℤ (posℤ→Int+Int≡+ zero m) ⟩
+      Int.predℤ (ℤ→Int ((neg zero) + m))
+        ≡⟨ predℤ→Int ((neg zero) + m) ⟩
+      ℤ→Int (predℤ ((neg zero) + m))
+        ≡⟨ cong ℤ→Int (predℤ-+ˡ (neg zero) m) ⟩
+      ℤ→Int ((neg (suc zero)) +  m) ∎
+    negsucℤ→Int+Int≡+ (suc n) m =
+      Int.negsuc (suc n) Int.+ ℤ→Int m
+        ≡⟨ sym (Int.predℤ+ (Int.negsuc n) (ℤ→Int m)) ⟩
+      Int.predℤ ((Int.negsuc n) Int.+ ℤ→Int m)
+        ≡⟨ cong Int.predℤ (negsucℤ→Int+Int≡+ n m) ⟩
+      Int.predℤ (ℤ→Int ((neg (suc n)) + m))
+        ≡⟨ predℤ→Int ((neg (suc n)) + m) ⟩
+      ℤ→Int (predℤ ((neg (suc n)) + m))
+        ≡⟨ cong ℤ→Int (predℤ-+ˡ (neg (suc n)) m) ⟩
+      ℤ→Int ((neg (suc (suc n))) + m) ∎
+
+  +'≡+ : _+'_ ≡ _+_
+  +'≡+ =
+    _+'_
+      ≡⟨ cong ( _∘_ (λ f → Int→ℤ ∘ f)) (funExt₂ ℤ→Int+Int≡+) ⟩
+    (λ n → (λ m → (Int→ℤ (ℤ→Int (n + m)))))
+      ≡⟨ funExt₂ (λ n m → (Iso.rightInv isoIntℤ (n + m))) ⟩
+    _+_ ∎
+
+  op≡Intℤ : (Int.ℤ → Int.ℤ → Int.ℤ) ≡ (ℤ → ℤ → ℤ)
+  op≡Intℤ = isoToPath (binaryOpIso isoIntℤ)
+
+  +Int≡+' : (λ i → op≡Intℤ i) [ Int._+_ ≡ _+'_ ]
+  +Int≡+' = transport-filler op≡Intℤ Int._+_
+
+
++Int≡+ : (λ i → (op≡Intℤ ∙ refl) i) [ Int._+_ ≡ _+_ ]
++Int≡+ = compPathP +Int≡+' +'≡+
```

Comment:
Can you explain why you want to export this particular form?

Reply:
My idea was that reasoning on equality is the simplest reasoning there is, so that's likely what other modules would want to use. In particular, if I want to show that two definitions of rationals using different definitions of integers as their base, result in actually equal types (which is the direct motivation for this work) then this form lets me directly show that specific parts of the definition are equal as integers.

Reply:
Ok, enough reason to export that.